### ISSUE TITLE
Could you share VGG-SS?

### ISSUE THREAD

**Issue by WindBlowMyAss** [denfed/heartheflow#1]
Hey, bro @denfed  Recently I start to do researches with Sound Source Localization and I have get the SoundNet-Flickr test set. However I can't obtain the VGG-SS, the steps in vgg's paper is tedious. So could you share that with me? Best wishes to you if you will, thanks!

**Comment by denfed** [comment]
Thanks for your interest! I personally can’t release the data for VGG-SS myself due to youtube’s licenses, rights, etc. Essentially, you need to manually download the training data for VGGSound and Flickr using download scripts that manually download videos, and preprocess them after. I’ll have detailed guides on how to download everything, but that part needs to be manually done. I will provide scripts to take those downloaded videos and preprocess them so that you can start training models right away (this part takes the longest to figure out how previous papers have done it). The code will be uploaded before new years. Thanks!

**Comment by WindBlowMyAss** [comment]
> Thanks for your interest! I personally can’t release the data for VGG-SS myself due to youtube’s licenses, rights, etc. Essentially, you need to manually download the training data for VGGSound and Flickr using download scripts that manually download videos, and preprocess them after. I’ll have detailed guides on how to download everything, but that part needs to be manually done. I will provide scripts to take those downloaded videos and preprocess them so that you can start training models right away (this part takes the longest to figure out how previous papers have done it). The code will be uploaded before new years. Thanks!

Thank you very much!
